test/e2e/localnet-underlay.go

```diff
@@ -117,6 +131,19 @@ func ovsEnableVLANAccessPort(ovsNodeName string, bridgeName string, portName str
 	return nil
 }
 
+func ovsRemoveVLANAccessPort(ovsNodeName string, bridgeName string, portName string, vlanID int) error {
+	cmd := []string{
+		"kubectl", "-n", ovnNamespace, "exec", ovsNodeName, "--",
+		"ovs-vsctl", "del-port", bridgeName, portName,
+	}
+
+	if _, err := runCommand(cmd...); err != nil {
+		return fmt.Errorf("failed to remove port %s to OVS bridge %s: %v", portName, bridgeName, err)
+	}
+
+	return nil
```

Comment:
we don't ned the vlanID to remove a port, in fact is not being used here.

Reply:
you're right. DONE